homomorphic/add.go

```diff
@@ -15,3 +20,18 @@ func Add(api frontend.API, a, b twistededwards.Point) (twistededwards.Point, err
 	curve.AssertIsOnCurve(b)
 	return curve.Add(a, b), nil
 }
+
+// AddPair returns the result (a.P1 + b.P1), (a.P2, b.P2) as a Pair
+func AddPair(api frontend.API, a, b Pair) (Pair, error) {
```

Comment:
why not wrap `Add` method and support variadic inputs instead of repeat the same logic for just 2 pairs?